src/ui-managers.ts

```diff
@@ -0,0 +1,26 @@
+import {BasePlugin} from 'kaltura-player-js';
+import {SidePanelsManager} from './services/side-panels/side-panels-manager';
+
+export const pluginName = 'uiManagers';
+export class UIManagersPlugin extends BasePlugin {
+  static defaultConfig = {};
+
+  constructor(name: string, player: any) {
+    super(name, player);
+
+    const sidePanelsManager: SidePanelsManager = new SidePanelsManager(player);
+    player.registerService('sidePanelsManager', sidePanelsManager);
+  }
+
+  reset(): void {
```

Comment:
why do you override it if it's empty ?

Reply:
Unfortunately i have no choice due to the linter
@typescript-eslint/no-empty-function

Reply:
idn

Reply:
done